src/cli.rs

```diff
@@ -385,19 +391,20 @@ impl Cli {
             cli.request_items.push(request_item.parse()?);
         }
 
-        if matches!(
-            app.get_bin_name().and_then(|name| name.split('.').next()),
-            Some("https") | Some("xhs") | Some("xhttps")
-        ) {
+        let bin_name = app.get_bin_name().and_then(|name| name.split('.').next());
+        if matches!(bin_name, Some("https") | Some("xhs") | Some("xhttps")) {
             cli.https = true;
         }
+        if matches!(bin_name, Some("http") | Some("https")) {
+            cli.strict_compat_mode = true;
+        }
```

Comment:
Maybe I should also add an env for enabling `strict_compat_mode`? This could be helpful for testing purposes and for users who have installed `xh` via a package manager.

Reply:
Update: I have now added `XH_HTTPIE_COMPAT_MODE` env variable.